### ISSUE TITLE
bug: hugo latest release has no packages

### ISSUE THREAD

**Issue by philclifford** [wimpysworld/deb-get#1237]

**Comment by tapyu** [comment]
Maybe the following error is related to this issue:

```
❯ deb-get update
  [+] Updating /etc/deb-get/01-main
  -=O=-                #        #        #         #
  [+] Updating cache of supported apps in the background
Hit:1 http://security.ubuntu.com/ubuntu noble-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu noble InRelease
Hit:3 https://brave-browser-apt-release.s3.brave.com stable InRelease
Hit:4 https://download.docker.com/linux/ubuntu noble InRelease
Hit:5 https://download.sublimetext.com apt/stable/ InRelease
Hit:6 https://repo.protonvpn.com/debian stable InRelease
Get:7 https://zotero.retorque.re/file/apt-package-archive ./ InRelease [2,153 B]
Hit:8 http://archive.ubuntu.com/ubuntu noble-updates InRelease
Hit:9 https://packages.microsoft.com/repos/code stable InRelease
Hit:10 http://archive.ubuntu.com/ubuntu noble-backports InRelease
Hit:11 https://ppa.launchpadcontent.net/zhangsongcui3371/fastfetch/ubuntu noble InRelease
Get:12 https://apt.fury.io/wez * InRelease
Fetched 7,874 B in 2s (3,191 B/s)
Reading package lists...
[!] ERROR! Missing required information of github package hugo:
URL=
VERSION_PUBLISHED=
```

**Comment by flexiondotorg** [comment]
Fixed via #1236
